pkg/inspect/parse.go

```diff
@@ -81,20 +81,27 @@ func toSpecificArray(array reflect.Value, target reflect.Type) (reflect.Value, e
 	return result, nil
 }
 
-
 // SanitizeSecrets visits all nodes and returns copies of the struct with secrets that are not passthrough
-// replaced
-func SanitizeSecrets(ctx context.Context, i interface{}) (interface{}, error) {
-	t, err := sanitizeSecretsReflect(ctx, reflect.ValueOf(i), secretHandler)
+// replaced. Regular token are replaced, files are replaced with the file name prefixed by relativeSecretPath
+// e.g.
+// a.b: encrypted:xxx -> a.b: <decrypted value>
+// a.b: encryptedFile:xxx -> a.b: relativeSecretPath + temp file name used
+func SanitizeSecrets(ctx context.Context, relativeSecretPath string, i interface{}) (interface{}, error) {
```

Comment:
How is this different than decrypting all secrets in a struct? Maybe a better name would be `DecryptSecrets` instead of `SanitizeSecrets`?

Reply:
It has changed in the next PR.